src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory instance;
+
+    private HibernateUtil() {
+    }
+
+    public static SessionFactory getSessionFactory() {
+        if (instance == null) {
```

Comment:
in which cases your instance won't to be null?

Reply:
my mistake)))